numba/cuda/cudadrv/driver.py

```diff
@@ -223,7 +223,10 @@ def __init__(self):
             self.is_initialized = True
             self.initialization_error = e
```

Comment:
Still seeing the same issue because of this line (when I comment it out the issue goes away).

Reply:
Presumably `.initialize()` is never exercised in the scenario I'm testing (which is simply to `import datashader` on a machine without CUDA)

Reply:
Argh! Sorry I missed this, will fix up tomorrow.

Reply:
@philippjfr This should now be resolved - could you let me know if it's fixed up all the issues in your test please?

Reply:
Can confirm, fixed now.

Reply:
Many thanks for re-testing!